src/commands/validate.js

```diff
@@ -3,9 +3,31 @@ const colors = require('colors/safe');
 
 const utils = require('../utils');
 
+// shadow of engine/style_checker/_condense_issues
```

Comment:
Keep in mind this is a public repo, so references to other Zapier repos can be confusing. This one is harmless, but I think it's worth while for us to practice a mindset of separating concerns.